var/spack/repos/builtin/packages/py-antspyx/package.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAntspyx(PythonPackage):
+    """Advanced Normalization Tools in Python."""
+
+    homepage = "https://pypi.org/project/antspyx/"
+    url = "https://github.com/ANTsX/ANTsPy/archive/refs/tags/v0.2.7.tar.gz"
+
+    version("0.2.7", sha256="495868dcb975486203cd1ce901c803e4b5d71fad5ad5c2525612de8e030f6a34")
+    version("0.2.4", sha256="357d9f93fdac8ca76f660d23f97239a5949284664866f8ba254b912afa953e55")
```

Comment:
One patch is obsolete, I've just updated to the newest version for simplicity.

There's the patched script that gets called during the installation process, I think it would be safest to use the resources rather than letting `antspy` download its own dependencies?

Reply:
Yes, resources are best so that the build works on air-gapped networks.